dev/tests/integration/testsuite/Magento/Customer/Block/Form/RegisterTest.php

```diff
@@ -12,6 +12,7 @@
 /**
  * Test class for \Magento\Customer\Block\Form\Register
  *
+ * @codingStandardsIgnoreFile
```

Comment:
Please remove the annotation.
It causes exception during Integration tests execution.